hw_9_2.ipynb

```diff
@@ -0,0 +1,263 @@
+{
```

Comment:
<p>you have to add what you actually want to do. What sensor are you using, what do they measure, etc ...</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>